spec/ruby/core/time/new_spec.rb

```diff
@@ -523,9 +523,16 @@ def obj.to_int; 3; end
       end
 
       it "raise TypeError is can't convert precision keyword argument into Integer" do
+        error_msg =
+          if "3.3" <= RUBY_VERSION
+            "no implicit conversion of String into Integer"
+          else
+            "no implicit conversion from string"
+          end
```

Comment:
I think a regexp would be simpler here to just allow both

Reply:
You're right, but regex would allow a regression and making it explicit means that we can remove this code once we drop support for Ruby 3.2 in the future.

Reply:
Right. I'll push a commit to use a `ruby_version_is "3.3"` then to make it clear it changed on purpose.
It's definitely good to be consistent in error messages, otherwise it's really hard for alternative Rubies to match them, and it makes harder for end users to test error cases.